source/trx2junit/Program.cs

```diff
@@ -15,16 +15,19 @@
 
 try
 {
-    Worker worker         = new();
-    WorkerOptions options = WorkerOptions.Parse(args);
+    Worker worker                   = new();
+    worker.WorkerNotification      += static (s, e) => Console.WriteLine(e.Message);
```

Comment:
`Console.WriteLine` should be put under the same lock as the error handler to avoid potential races.